### DISCUSSION TITLE
WinUI3 app throws NullRef at build time after switching to WindowEx

### DISCUSSION THREAD

**Discussion by easyrhino-gh** [dotMorten/WinUIEx#202] [Q&A]
Title pretty much says it all. The exception is thrown in the generated .cs file for MainWindow which I have switched over to WindowEx. I have the sample project for [XamlBrewer's XamlBehaviors sample](https://github.com/XamlBrewer/XamlBrewer.WinUI3.XamlBehaviors.Sample) and I can't find any differences.
TIA for any assistance!

**Comment by dotMorten** [comment]
Thanks but the title doesn't say it all. Can you please provide at the very least a callstack, and even better a reproducer?
